### ISSUE TITLE
Installing `flex` fails since `help2man` is not available

### ISSUE THREAD

**Issue by eschnett** [LLNL/spack#2427]
Installing `flex` fails for me on an Ubuntu 14.04 system where `help2man` is not installed on the system. Configuring and installing `flex` output this:

```
checking for help2man... ${top_srcdir}/build-aux/missing help2man
configure: WARNING: program not found: help2man: building man page will not work
```
and later
```
 /usr/bin/install -c -m 644 ./flex.1 '/xfs1/eschnetter/src/spack/opt/spack/linux-ubuntu14-x86_64/gcc-6.2.0/flex-2.6.2-vjjx5kmkefarm3ii4k6mdtrns6v77fqg/share/man/man1'
/usr/bin/install: cannot stat ‘./flex.1’: No such file or directory
make[2]: *** [install-man1] Error 1
```

It seems that `install` requires the man page to be present. `flex` should probably depend on `help2man`, but this doesn't exist in Spack yet.

**Comment by adamjstewart** [comment]
Already fixed in #2401.

**Comment by tgamblin** [comment]
Fixed by #2401
